packages/slider/src/Slider.js

```diff
@@ -123,70 +113,51 @@ class Slider extends Component<Props, State> {
       sliderPos,
       direction
     });
-  }
+  };
 
-  handleNoop = (e) => {
+  handleNoop = e => {
     e.stopPropagation();
     e.preventDefault();
-  }
+  };
 
   onChange = (start, end) => {
-    let {
-      onChange
-    } = this.props;
+    let { onChange } = this.props;
 
     this.setState({
-      start, end
+      start,
+      end
     });
 
     if (!onChange) return;
     onChange(start, end);
-  }
+  };
 
-  getPositionFromValue = (value) => {
-    const {limit} = this.state;
-    const {min, max} = this.props;
+  getPositionFromValue = value => {
+    const { limit } = this.state;
+    const { min, max } = this.props;
     const diffMaxMin = max - min;
     const diffValMin = value - min;
     const percentage = diffValMin / diffMaxMin;
 
     return Math.round(percentage * limit);
-  }
+  };
 
   render() {
-    let {
-      id,
-      label,
-      min,
-      max,
-      step,
-      unit,
-      minRange,
-      tooltips,
-      range,
-      fixed,
-    } = this.props;
-
-    let {
-      sliderPos,
-      direction,
-      start,
-      end
-    } = this.state;
+    let { id, label, min, max, step, unit, minRange, tooltips, range, fixed } = this.props;
+
+    let { sliderPos, direction, start, end } = this.state;
 
     return (
       <div id={id}>
         {label && <InputLabel htmlFor={id}>{label}</InputLabel>}
-        <div className="m-range-slider" ref={(s) => {
-          this.slider = s
-        }}>
+        <div
+          className="m-range-slider"
+          ref={s => {
+            this.slider = s;
+          }}
+        >
           <div className="m-range-slider__inner">
-            <Bar
-              start={start}
-              end={end}
-              getPositionFromValue={this.getPositionFromValue}
-              range={range}
-            />
+            <Bar start={start} end={end} getPositionFromValue={this.getPositionFromValue} range={range} />
```

Comment:
Why not keeping the more readable format?

Reply:
Er zit geen formatering configs ( prettiers, eslint, editorconfig, ... ) in de repo. Ik heb het wel aangepast en issue van gemaakt.